scripts/infra/local-setup.sh

```diff
@@ -2,6 +2,10 @@
 # SPDX-License-Identifier: Apache-2.0
 
 rh__ensure_ibm_system_pkgs() {
+    if command -v ibmcloud &> /dev/null; then
+        echo "ibmcloud CLI is installed"
```

Comment:
nit: should this say "is already installed" so it's clear the script didn't just install it?